Makefile.am

```diff
@@ -26,13 +26,13 @@ include/mavlink/ardupilotmega/mavlink.h: modules/mavlink/pymavlink/tools/mavgen.
 		--wire-protocol 2.0 \
 		$(srcdir)/modules/mavlink/message_definitions/v1.0/ardupilotmega.xml
 
-AM_CPPFLAGS = \
```

Comment:
friends don't let friends use recursive makefiles on autotools. And now that you are depending on files from other dirs, it has the potential to break parallel build. Rather than doing this, just move things to a single Makefile.am